modeler.js

```diff
@@ -33,45 +35,69 @@ async function createProject([projectName]) {
   };
 
   try {
-    // Call the add endpoint.
-    const response = await axios(options);
+    // Call the add project endpoint.
+    const response = await axios(projectOptions);
 
-    // Process the results from the API call.
+    // Capture data from the new project.
     const newProject = response.data;
 
     console.log(
       `Project added! Name: ${newProject.name}. ID: ${newProject.id}.`
     );
+
+    // Step 2: Add a collaborator to the project.
+
+    // This is the API endpoint to add a collaborator to a project.
+    const collaboratorUrl = `${modelerApiUrl}/collaborators`;
+
+    // Configure the API call.
+    const collaboratorOptions = {
+      method: "PUT",
+      url: collaboratorUrl,
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${accessToken}`
+      },
+      data: {
+        // The body contains information about the project and collaborator.
+        email: adminEmail,
+        projectId: newProject.id,
+        role: "project_admin"
+      }
+    };
+
+    // Call the add collaborator endpoint.
+    const collaboratorResponse = await axios(collaboratorOptions);
+
+    if (collaboratorResponse.status === 204) {
+      console.log(`Collaborator added! Email: ${adminEmail}.`);
+    } else {
+      console.error("Unable to add collaborator!");
+    }
   } catch (error) {
     // Emit an error from the server.
     console.error(error.message);
   }
 }
 
-// A function that adds a new collaborator to a project
-async function addCollaborator([collaboratorEmail, projectId, role]) {
+// An action to view details of a project.
```

Comment:
This is very nit-picky, but I see we shift between "a project" and "the project." For the scope of this tutorial, I _think_ we just mean "the project," right? As in _the_ project where our model sits when we are working with as a solo collaborator/admin.

Reply:
I see a distinction between the use of `the` and `a`.

In the previous action, and in line 111 (which I just updated because it was out of date), we refer to _the_ project because it's a specific project that we have in memory. 

In this comment, we're describing that this action can be used to view _any_ project, based on the argument passed in.